docs/content/en/docs/_index.md

```diff
@@ -19,7 +19,7 @@ cOS and derivatives shares a common feature set, can be upgraded in a **OTA-alik
 cOS comes also with vanilla images that can be used to boot directly container images built with the toolkit.
 
 {{% alert title="Note" %}}
-Note here we refer to "OTA-alike" merely as a method of distributing upgrades. It does not involve any specific setup (either wireless or cabled works) except an network connection to the registry where images are stored. Updates are delivered from one central location (the container registry) which hosts all images available for the clients to pull from.
+Note here we refer to "OTA-alike" merely as a method of distributing upgrades. It does not involve any specific setup (either wireless or cabled works) except an network connection to the registry where images are stored. Updates are delivered from one central location (the container registry) which hosts all images available for the clients to pull from in A/B fashion.
```

Comment:
should we just drop the OTA-alike directly and just use A/B system?

Reply:
good point, done :+1: